script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -98,7 +98,7 @@ I'm already used to it at this point, so I'm not particularly surprised.
 -- Page 582, Offset 18031.
 -- もー、志貴ったらつまらなそうな顔してるクセに、反応だけはいいんだからー！」
 Geez, you've got such a bored looking expression on your face,
- but at least that response showed some enthusiasm!"
+ but at the very least that response showed some enthusiasm!"
```

Comment:
Not sure the extra qualifier works here

Reply:
I think my original thought was to maybe add "well" after "at" but then that would probably warrant another comma which this line already had plenty of, and so i went for the next best, yet insuffcient: "the very".

Wanted to add "Well" as Arc being like "Well you know at least that shows some enthusiasm". You reckon "well" wouldn't have worked in the first place?

Reply:
original is better